frontend/src/metabase/components/Schedule/Schedule.tsx

```diff
@@ -85,6 +78,18 @@ export const Schedule = ({
     change: ScheduleChangeProp,
   ) => void;
 }) => {
+  /** Fill out schedule template with components */
+  const fill = useCallback(
+    (template: string, components: ReactNode[]) =>
+      fillScheduleTemplate(
+        template,
+        components,
+        // Memoization function created here so it gets garbage collected when the Schedule component is unmounted
+        _.memoize(getStringWidth),
+      ),
+    [],
+  );
```

Comment:
@sloansparger This is my attempt to implement your suggestion relating to memoization and gc